src/data-table.component.ts

```diff
@@ -302,6 +302,12 @@ export class DataTableComponent implements OnInit, OnDestroy, OnChanges {
    */
   @Input() public extensions: TemplateRef<any>[] = [];
 
+  /**
+   *  Flag to use external column menu
+   */
+
+  @Input() public hasExtColMenu: boolean = false;
```

Comment:
@jozzyka please add this prop's description to the README (`Table settings` section)

Reply:
@jozzyka please add the changes to the `README.adoc` the `README.md` is generated